python/lgbserver/lgbserver/__main__.py

```diff
@@ -51,4 +51,4 @@
         model_repository = LightGBMModelRepository(args.model_dir, args.nthread)
         # LightGBM doesn't support multi-process, so the number of http server workers should be 1.
         server = kserve.ModelServer(workers=1, registered_models=model_repository)
-        server.start([model] if model.ready else [])
+        server.start([model])
```

Comment:
```suggestion
        server.start([])
```

Reply:
also line 45 should be changed to 
`kserve.ModelServer(workers=1).start([model])`